tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
   // see https://www.typescriptlang.org/tsconfig to better understand tsconfigs
   "include": ["src", "types"],
-  "exclude": ["__mocks__"],
+  "exclude": ["__mocks__", "*.documentation.ts"],
```

Comment:
Why do we not want to use the `__documentation__` director name convention? Or even `.stories` for files? Or `__stories__` for directories. We don't want anything exported from those directories ever included in our builds, so we should be sure to exclude them all, right?

Reply:
we can, but that's more of a chore and I'm just trying to get things working right now. getting closer!